### PR TITLE
Update OneNote Mimetype.

### PR DESCRIPTION
**Pull request by njohner**
We include application/msonenote as mimetype for OneNote.

For [CA-2674]

## Checklist

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

[CA-2674]: https://4teamwork.atlassian.net/browse/CA-2674

### CODE REVIEW

File: changes/CA-2674.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Update OneOffixx Mimetype. [njohner]
```
Comment (phgross) [inline changes/CA-2674.bugfix:1]: Should be onenote not oneoffixx.

File: opengever/core/upgrades/20210802134316_update_one_offixx_mimetype/upgrade.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from ftw.upgrade import UpgradeStep
+
+ONEOFFIXX_EXTENSIONS = ['.one']
+
+
+class UpdateOneOffixxMimetype(UpgradeStep):
```
Comment (phgross) [inline opengever/core/upgrades/20210802134316_update_one_offixx_mimetype/upgrade.py:6]: OneNote

File: opengever/core/upgrades/20210802134316_update_one_offixx_mimetype/upgrade.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+from ftw.upgrade import UpgradeStep
+
+ONEOFFIXX_EXTENSIONS = ['.one']
```
Comment (phgross) [inline opengever/core/upgrades/20210802134316_update_one_offixx_mimetype/upgrade.py:3]: ONENOTE

**Review by phgross — CHANGES_REQUESTED** [review]

File: changes/CA-2674.bugfix (hunk 1 above)
Comment (njohner) [reply changes/CA-2674.bugfix:1]: 😅 sorry...

**Comment by njohner** [comment]
should be better now

**Pull request merged**